app/Controllers/LibriController.php

```
@@ -2691,12 +2691,138 @@ public function generateLabelPDF(Request $request, Response $response, mysqli $d
+                $this->renderPortraitLabel($pdf, $appName, $libro, $autoriStr, $collocazione, $barcodePayload, $positionText, $availableWidth, $availableHeight, $margin, $numeroInventario);
+            } else {
+                $this->renderLandscapeLabel($pdf, $appName, $libro, $autoriStr, $collocazione, $barcodePayload, $positionText, $availableWidth, $availableHeight, $margin, $numeroInventario);
+            }
+        }
+
+        $pdfContent = $pdf->Output('', 'S');
+        $response->getBody()->write($pdfContent);
+        return $response
+            ->withHeader('Content-Type', 'application/pdf')
+            ->withHeader('Content-Disposition', 'inline; filename="etichette_copie_' . $id . '.pdf"');
+    }
```

PHPMD (2.15.0)

[error] Avoid assigning values to variables in if clauses and the like (line '2757', column '17'). (undefined)

(IfStatementAssignment)